test/integration/StorageBackends.spec.js

```diff
@@ -75,4 +96,16 @@ describe('Storage Features Test', function () {
         }
         return 1;
     }
+
+    async function verifyStreamContent(inputStream) {
```

Comment:
This should be possible without writing to a file. Can't you just collect it in memory and check it?

Reply:
This method now uses a `StringWritable` pipeline.